### ISSUE TITLE
The Name property of 'Function input' list must not include the element's control type.

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1676]
Repro steps:
Open the Application Calculator
Navigate to ‘Graphing’ button and activate it.
Now add some equation and Run accessibility Insight for Windows. 
Observe the issue.

Actual Result:
The Name property of the equation include the element's control type. Name is defined as 'Function input list'.

Expected Result:
The Name property of the equation must not include the element's control type. Name should be defined correctly as 'Function input' and should not include role 'list' in the name.

**Comment by mateuszplaza** [comment]
Working on this!
